tests/test_computer.py

```diff
@@ -150,9 +155,48 @@ def test_dynamic_info_firecrest_version(firecrest_config, monkeypatch, tmpdir: P
     }
 
     # should catch FIRECREST_VERSION if value is not provided
-    result = _dynamic_info_firecrest_version(ctx, None, "0")
+    result = _dynamic_info_firecrest_version(ctx, None, "None")
     assert isinstance(result, str)
 
     # should use the value if provided
     result = _dynamic_info_firecrest_version(ctx, None, "10.10.10")
     assert result == "10.10.10"
+
+    # raise BadParameter if the version is not supported
+    with pytest.raises(BadParameter) as exc_info:
```

Comment:
pytest has an argument to match the error message
```suggestion
    with pytest.raises(BadParameter, match=r".*FirecREST api version 0.0.0 is not supported.*")):
```